src/views/document/utils/boxes/MapBox.vue

```diff
@@ -142,13 +142,22 @@ export default {
 
       feature.set('name', name);
 
-      const filename = this.document.document_id + extension;
+      const filename = this.getDownloadedFileName(name, extension);
       const content = format.writeFeatures([feature], {
         featureProjection: 'EPSG:3857',
       });
 
       utils.download(content, filename, mimetype + ';charset=utf-8');
     },
+
+    getDownloadedFileName(name, extension) {
+      const prefixRouteId = name.length < 100 ? this.document.document_id + '_' : '';
```

Comment:
That would be more consistent to always prefix with the route id but substring the whole name at the first 100 characters if really to long?

Reply:
`prefixDocumentId` would be more relevant as the document may be an outing :P

Reply:
> That would be more consistent to always prefix with the route id but substring the whole name at the first 100 characters if really to long?

it would be better indeed